mantidproject/mantid
# Remove BOM from all python files.

Python 3 is having problems importing files with the [BOM](https://en.wikipedia.org/wiki/Byte_order_mark), giving `Error importing module '...': invalid character in identifier`. We don't need this magic


**To test:**
You should now be able to run the system tests without seeing the `invalid character in identifier` error. Run something like `./systemtest -R DOSTest` and check that there are fewer `Error importing module` at the start.

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
